.github/workflows/init.yml

```diff
@@ -76,8 +80,17 @@ jobs:
             COLLECTOR_TAG="$(make tag)"
           fi
 
-          echo "collector-tag=${COLLECTOR_TAG}" >> "$GITHUB_OUTPUT"
-          echo "collector-image=quay.io/stackrox-io/collector:${COLLECTOR_TAG}" >> "$GITHUB_OUTPUT"
+          COLLECTOR_TEST_TAG="$COLLECTOR_TAG"
+          if [[ '${{ contains(github.event.pull_request.labels.*.name, 'build-full-images') }}' == 'true' ]]; then
+            COLLECTOR_TEST_TAG="$COLLECTOR_TAG-latest"
+          fi
+
+          {
+            echo "collector-tag=${COLLECTOR_TAG}"
+            echo "collector-image=quay.io/stackrox-io/collector:${COLLECTOR_TAG}"
+            echo "collector-test-tag=${COLLECTOR_TEST_TAG}"
+          } >> "$GITHUB_OUTPUT"
```

Comment:
Why do we need this tag? If the `build-full-images` label is set CI builds the full `quay.io/stackrox-io/collector:${COLLECTOR_TAG}` and if it is not set the `-slim` variation gets re-tagged to this same image, that makes it so the integration tests can always use the same image to run in either case.

See:
- https://github.com/stackrox/collector/blob/a61c6aed1ff3178b5d55d282ff4e139a94851180/.github/workflows/collector-full.yml#L117-L160
- https://github.com/stackrox/collector/blob/a61c6aed1ff3178b5d55d282ff4e139a94851180/.github/workflows/collector-full.yml#L34-L36

Reply:
In my testing this was not working and the `quay.io/stackrox-io/collector:${COLLECTOR_TAG}` image did not contain the drivers. I'll investigate further.